pkg/apis/apps/validation/validation_test.go

```diff
@@ -42,6 +42,96 @@ func intStrAddr(intOrStr intstr.IntOrString) *intstr.IntOrString {
 	return &intOrStr
 }
 
+type setTweak func(s apps.StatefulSet)
+
+func makeSet(selectorLabels *map[string]string, template *api.PodTemplate, tweaks ...setTweak) apps.StatefulSet {
```

Comment:
`makeStatefulSet` or `mkStatefulSet` seem better - "Set" could be a replicase, daemonset, statefulset, ...

Reply:
modified as suggested